templates/weblogic.service.j2

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Weblogic Service for nDelius
+After=network.target
+
+[Service]
+Type=simple
+{% if wls_service_user is defined and wls_service_user.name is defined %}
+User={{ wls_service_user.name }}
```

Comment:
you can default the user so `User={{ wls_service_user.name|default('oracle') }}` will do the trick here

Reply:
Ansible-playbook was crashing when i did it that way, as the wls_service_user.name was calling .name on a null argument.

Reply:
Ok fair enough, it shouldn't have, but lets leave this as is then